app/controllers/hosts_controller.rb

```diff
@@ -342,15 +343,7 @@ def ipmi_boot
   def console
     return unless @host.compute_resource
     @console = @host.compute_resource.console @host.uuid
-    @encrypt = Setting[:websockets_encrypt]
-    render case @console[:type]
-             when 'spice'
-               "hosts/console/spice"
-             when 'vnc'
-               "hosts/console/vnc"
-             else
-               "hosts/console/log"
-           end
+    super
```

Comment:
should it be better to alias method chain instead of super?

Reply:
I don't think so. Isn't that, what `super` was made for? Calling a method "lower" on the inheritance chain? Mind explaining, why you'd prefer `alias_method_chain`?